src/pages/resultsView/survival/SurvivalChart.tsx

```diff
@@ -137,13 +137,11 @@ export default class SurvivalChart extends React.Component<ISurvivalChartProps,
 
         return (
 
-            <div>
+            <div className="posRelative">
 
+                <div className="borderedChart">
 
-
-                <div className="borderedChart" style={{position:'relative'}}>
-
-                    <div className="btn-group" style={{position:'absolute', right:10 }} role="group">
+                    <div className="btn-group" style={{position:'absolute', zIndex:10, right:10 }} role="group">
```

Comment:
it would have been nice if we had a test to detect this failure.  because it's css issue, I think the only possiblity would e2e test.  Even then it would be hard because the test would have had to use positioning to click rather than $(el).click in order to be subject to the zindex issue.